appeals/e2e/cypress/page_objects/caseDetailsPage.js

```diff
@@ -134,6 +134,16 @@ export class CaseDetailsPage extends Page {
 		this.elements.manageAgreementToChangeDescriptionEvidence().click();
 	}
 
+	verifyTableCellTextCaseHistory(answer) {
```

Comment:
This new method should go into the "Verfifications" area of the file, so next to `verifyAnswerSummaryValue()` a little further down

Reply:
good point - thank you, have moved in second commit